### ISSUE TITLE
orientation auto issue on iOS

### ISSUE THREAD

**Issue by stharvey** [plepe/Leaflet.TextPath#4]
Thanks for the work on this plugin, it's great.

However, on iOS (only tested on 15.7 iPhone 6s) the letters appear upsidedown with orientation auto: 

![IMG_0063.jpg](https://user-images.githubusercontent.com/23450702/210281551-4573f2e0-3334-4b17-9c67-321fd7219b15.jpg)

It works no problem on Android:
![Screenshot_20230102-220437.png](https://user-images.githubusercontent.com/23450702/210281563-3ca1e5f4-f0e3-490a-8e03-71cceb9f7c5f.png)

Do you know why this might be happening?

**Comment by plepe** [comment]
Yes, I know. Unfortunately, the library I'm using for labels can't handle automatic orientation, so this appears on all platforms.

A few years ago I made an effort to implement this, and I actually got quite far. Unfortunately I had to abandon it due to browser incompatibilities. I should look into it again, if support is better now:
[makinacorpus/Leaflet.TextPath#72](https://github.com/makinacorpus/Leaflet.TextPath/pull/72)

On the other hand, I'd like to get rid of this textpath library altogether, because it uses SVG rendering, which slow browsers down badly (alternative: canvas rendering).

**Comment by plepe** [comment]
This is a branch of the library [Leaflet.TextPath](https://github.com/makinacorpus/Leaflet.TextPath), where I tried to handle automatic orientation, and it somehow worked in some browsers. The code is still there, but I think it will never mature.

I finally had to abandon it due to browser incompatibilities. You can read more about that in the [original pull request](https://github.com/makinacorpus/Leaflet.TextPath/pull/72).

You can check the SVG support of your browser on this link. Please send me a screenshot of the first image (and tell me browser and OS). You can see, that these images look very different (as far as I can tell, nothing has changed during the last 4 years).

I'd like to get rid of this textpath library altogether, because it uses SVG rendering, which slow browsers down badly (alternative: canvas rendering).
